### PR TITLE
Added Described Provisioned Product operation.

### PR DESCRIPTION
**Pull request by LenutaToderean**
Added Described Provisioned Product operation. - Java Action

### CODE REVIEW

File: cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/servicecatalog/DescribeProvisionedProductAction.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/*
+ * (c) Copyright 2017 EntIT Software LLC, a Micro Focus company, L.P.
```
Comment (lrevnic) [inline cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/servicecatalog/DescribeProvisionedProductAction.java]: Should be 2018 Micro Focus

File: cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/servicecatalog/DescribeProvisionedProductAction.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                                       @Param(value = ASYNC, description = ASYNC_DESC) String async,
+                                       @Param(value = REGION, description = REGION_DESC) String region,
+                                       @Param(value = ACCEPTED_LANGUAGE, description = PROVISIONED_PRODUCT_ACCEPTED_LANGUAGE_DESCRIPTION) String acceptedLanguage,
+                                       @Param(value = PRODUCT_ID, required = true, description = PROVISIONED_PRODUCT_ID_DESCRIPTION) String productId) {
+
+        //Assign default values to inputs
+        final String proxyPortVal = defaultIfEmpty(proxyPort, DefaultValues.PROXY_PORT);
+        final String connectTimeoutVal = defaultIfEmpty(connectTimeout, DefaultValues.CONNECT_TIMEOUT);
+        final String execTimeoutVal = defaultIfEmpty(execTimeout, DefaultValues.EXEC_TIMEOUT);
+        final String asyncVal = defaultIfEmpty(async, DefaultValues.ASYNC);
```
Comment (lrevnic) [inline cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/servicecatalog/DescribeProvisionedProductAction.java]: Default SYNC

File: cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/servicecatalog/DescribeProvisionedProductAction.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                                       @Param(value = PROXY_USERNAME, description = PROXY_USERNAME_DESC) final String proxyUsername,
+                                       @Param(value = PROXY_PASSWORD, encrypted = true, description = PROXY_PASSWORD_DESC) final String proxyPassword,
+                                       @Param(value = CONNECT_TIMEOUT, description = CONNECT_TIMEOUT_DESC) String connectTimeout,
+                                       @Param(value = EXECUTION_TIMEOUT, description = EXECUTION_TIMEOUT_DESC) String execTimeout,
+                                       @Param(value = ASYNC, description = ASYNC_DESC) String async,
+                                       @Param(value = REGION, description = REGION_DESC) String region,
+                                       @Param(value = ACCEPTED_LANGUAGE, description = PROVISIONED_PRODUCT_ACCEPTED_LANGUAGE_DESCRIPTION) String acceptedLanguage,
+                                       @Param(value = PRODUCT_ID, required = true, description = PROVISIONED_PRODUCT_ID_DESCRIPTION) String productId) {
+
+        //Assign default values to inputs
```
Comment (lrevnic) [inline cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/servicecatalog/DescribeProvisionedProductAction.java]: No default values for REGION and LANGUAGE?

File: cs-amazon/src/main/java/io/cloudslang/content/amazon/entities/constants/Descriptions.java [hunk 4]
```diff
@@ -100,4 +100,26 @@
...
+    public static class DescribeProvisionedProductAction {
+        //Inputs
+        public static final String DESCRIBE_PROVISIONED_PRODUCT_DESCRIPTION = "Gets information about the specified provisioned product.";
+        public static final String PROVISIONED_PRODUCT_ACCEPTED_LANGUAGE_DESCRIPTION = "The language code.\n" +
+                "Example: en (English), jp (Japanese), zh(Chinese)" +
+                "Default: 'en'";
+        public static final String PROVISIONED_PRODUCT_ID_DESCRIPTION = "The provisioned product identifier.";
+        public static final String PROVISIONED_PRODUCT_ARN_DESC = "The ARN of the provisioned product.";
+        public static final String PROVISIONED_PRODUCT_CREATED_TIME_DESC = "The UTC time stamp of the creation time.";
+        public static final String PROVISIONED_PRODUCT_ID_DESC = "The identifier of the provisioned product.";
```
Comment (lrevnic) [inline cs-amazon/src/main/java/io/cloudslang/content/amazon/entities/constants/Descriptions.java]: Can you add an example?

File: cs-amazon/src/main/java/io/cloudslang/content/amazon/entities/constants/Descriptions.java [hunk 5]
```diff
@@ -100,4 +100,26 @@
...
     }
+
+    public static class DescribeProvisionedProductAction {
+        //Inputs
+        public static final String DESCRIBE_PROVISIONED_PRODUCT_DESCRIPTION = "Gets information about the specified provisioned product.";
+        public static final String PROVISIONED_PRODUCT_ACCEPTED_LANGUAGE_DESCRIPTION = "The language code.\n" +
+                "Example: en (English), jp (Japanese), zh(Chinese)" +
+                "Default: 'en'";
+        public static final String PROVISIONED_PRODUCT_ID_DESCRIPTION = "The provisioned product identifier.";
+        public static final String PROVISIONED_PRODUCT_ARN_DESC = "The ARN of the provisioned product.";
```
Comment (lrevnic) [inline cs-amazon/src/main/java/io/cloudslang/content/amazon/entities/constants/Descriptions.java]: Can you add an example?

File: cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/servicecatalog/DescribeProvisionedProductAction.java (hunk 2 above)
Comment (LenutaToderean) [reply cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/servicecatalog/DescribeProvisionedProductAction.java]: ASYNC = false in DefaultValues.class.

**Pull request merged**